interactors/multi-column-list.js

```diff
@@ -13,7 +22,9 @@ export const MultiColumnListCell = HTML.extend('multi column list cell')
     column: (el) => el.textContent,
     columnIndex: childIndex,
     selected: (el) => !!el.parentElement.className.match(/mclSelected/),
-    measured: (el) => el.style && el.style.width !== ''
+    measured: (el) => el.style && el.style.width !== '',
+    visible: (el) => isVisible(el),
+    clientTop: el => el.getBoundingClientRect().top,
```

Comment:
It's always an art with interactors whether to include very render-specific properties. In fact, we've found a fundamental tension in other projects between interactors filters useful to the author of the component-library and design system interactor filters that are useful to application developers. It seems like the former is a superset of the latter. More simply, it's a question whether in any stripes apps, that you would need the `clientTop` property, but it is very useful in testing `stripes-components` itself. If an app developer would never use it, then its presence in the API can be confusing.

One way to relieve this tension is to make an `element` filter in your base interactor that exposes the underlying element. This isn't that useful by itself since you rarely know the element ahead of time (after all that's kinda what interactors are for). However, this does allow you use [matchers][matchers] implemented locally within your test suite against your element which has two implications:

1. You don't need to add render-specific filters that nobody will actually use outside of the component test suite.
2. You can use the matcher on _all_ of your interactors, not just the one that has the filter (in this case multi column list cell)

For example, let's say you had a matcher called `withClientTop`:

```js
export function withClientTop(value) {
  return {
    match(actual) {
      return actual.getBoundingClientRect().top === value;
    },
    description() {
      return `clientTop === ${value}`;
    }
  };
};
```

Now, you can use this anywhere:

```js
await MultiColumnListCell().has({ element: withClientTop(100) });
await Accordion().find(Button()).has({ element: withClientTop(55) });
```

If that's an option you'd want to take, then you could do it pretty easily by exporting your own `HTML` interactor from the index:

```js
import { HTML as BaseHTML } from '@bigtest/interactor';

export const HTML = BaseHTML
  .filters({
    element: element => element
  });
```

and having all the internal components use that as their base (consumers of stripes-testing would get this transparently since they already import `HTML` from folio, not bigtest)

[matchers]: https://frontside.com/bigtest/docs/interactors/matchers